torch/_inductor/triton_heuristics.py

```diff
@@ -825,22 +826,23 @@ def cached_autotune(
     if filename is not None and (len(configs) > 1 or config.coordinate_descent_tuning):
         cache_filename = os.path.splitext(filename)[0] + ".best_config"
         configs_hash = hash_configs(configs)
-        best_config = load_cached_autotuning(cache_filename, configs_hash, configs)
+        lock = FileLock(cache_filename + ".lock", mode=0o777)
```

Comment:
in `torch/_inductor/codecache` we use timeout.. use here as well? Do we need to set mode here, don't see in any of our other uses in the codebase